runtime/cudaq/platform/default/rest/helpers/quantinuum/quantinuum.config

```diff
@@ -15,11 +15,11 @@ GEN_TARGET_BACKEND=true
 # Add the rest-qpu library to the link list
 LINKLIBS="${LINKLIBS} -lcudaq-rest-qpu"
 
-# Define the lowering pipeline, here we lower to Base QIR
+# Define the lowering pipeline, here we lower to Adaptive QIR
 PLATFORM_LOWERING_CONFIG="expand-measurements,unrolling-pipeline,func.func(lower-to-cfg),canonicalize,func.func(quake-multicontrol-decomposition),quantinuum-gate-set-mapping"
 
 # Tell the rest-qpu that we are generating QIR.
-CODEGEN_EMISSION=qir-base
+CODEGEN_EMISSION=qir-adaptive
```

Comment:
Do we need a way for the user to select base profile for quantinuum?

Reply:
The Adaptive Profile is intended to be a superset of the Base Profile, so I don't think so. I'll mark this this as resolved, but feel free to re-open if you want to continue discussing.